ts/src/pro/cex.ts

```diff
@@ -174,9 +174,29 @@ export default class cex extends cexRest {
         const data = this.safeValue (message, 'data', []);
         const limit = this.safeInteger (this.options, 'tradesLimit', 1000);
         const stored = new ArrayCache (limit);
-        for (let i = 0; i < data.length; i++) {
+        // as described in watchTrades, only one symbol can be watched at a time
+        const subs = Object.keys (client.subscriptions);
+        if (!subs.length) {
+            return;
+        }
+        let symbol = undefined;
```

Comment:
unfortunately, we need to find out symbol, otherwise it fails down the road (being `null`) https://github.com/ccxt/ccxt/blob/21eef8e26279e94bf538b39ff9fa746db831003a/cs/ccxt/ws/ArrayCache.cs#L114 and etc